Nginx_Auth_Proxy/confd/templates/bichard.https.conf.tpl

```diff
@@ -88,12 +88,13 @@ location /bichard {
     auth_request /auth;
     auth_request_set $auth_cookie $upstream_http_set_cookie;
     add_header  Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
-    include /etc/includes/headers.conf;
+    include /etc/includes/ui-headers.conf;
     add_header Set-Cookie "$auth_cookie; secure";
 
     proxy_pass        https://$ui;
     proxy_ssl_verify  {{ getv "/cjse/nginx/proxysslverify" "on" }};
     proxy_set_header Host $http_host;
+    proxy_pass_header Content-Security-Policy;
```

Comment:
Adding the CSP to the response header is now handled by the UI itself

Reply:
I think a small comment explaining this change would be handy

Reply:
Done 💯